pmd-java/src/main/resources/category/java/security.xml

```diff
@@ -60,6 +60,34 @@ public class Foo {
         byte[] iv = "secret iv in here".getBytes();
     }
 }
+]]>
+        </example>
+    </rule>
+
+    <rule name="StaticFieldMustBeFinal"
+          language="java"
+          since="6.34.0"
```

Comment:
Just as a reminder - this currently needs to be `6.35.0` (that's the earliest version of PMD that contains the rule).